apps/src/applab/ApplabVisualizationColumn.jsx

```diff
@@ -1,9 +1,20 @@
+var Radium = require('radium');
 var Visualization = require('./Visualization');
 var GameButtons = require('../templates/GameButtons');
 var CompletionButton = require('./CompletionButton');
 var PlaySpaceHeader = require('./PlaySpaceHeader');
+var PhoneFrame = require('./PhoneFrame');
 var BelowVisualization = require('../templates/BelowVisualization');
+var ProtectedStatefulDiv = require('../templates/ProtectedStatefulDiv');
+var applabConstants = require('./constants');
```

Comment:
My feeling is that it's kind of weird having both next to each other. I didn't necessarily feel like converting this entire file, so I continued to use require.